csrankings-d.csv

```diff
@@ -943,6 +943,7 @@ Dexter Hadley,University of Central Florida,https://med.ucf.edu/person/dexter-ha
 Dexter Kozen,Cornell University,http://www.cs.cornell.edu/~kozen,LV1qWjgAAAAJ
 Deying Li,Renmin University of China,http://info.ruc.edu.cn/academic_professor.php?teacher_id=60,NOSCHOLARPAGE
 Dezhen Song,Texas A&M University,http://faculty.cs.tamu.edu/dzsong,jdSlREMAAAAJ
+Dezhong Yao 0002,HUST,http://faculty.hust.edu.cn/dezhong/en/index.htm,SFBHGr0AAAAJ

```

Comment:
Web page is unresponsive. Are you sure it's correct?

Reply:
> Web page is unresponsive. Are you sure it's correct?

Yes, the web page is correct, it responds correctly. please check again.